addons/l10n_in_edi/models/account_move.py

```diff
@@ -61,3 +62,14 @@ def _l10n_in_edi_is_managing_invoice_negative_lines_allowed(self):
         """
         param_name = 'l10n_in_edi.manage_invoice_negative_lines'
         return bool(self.env['ir.config_parameter'].sudo().get_param(param_name))
+
+    @api.depends('move_type')
```

Comment:
```suggestion
    @api.depends('move_type', 'line_ids.tax_tag_ids', 'amount_residual', 'partner_id.l10n_in_pan')
```
I don't know why this compute is firing so much, but if I change partner it is not updating.